tests/test_port_add_remove.py

```diff
@@ -119,7 +119,7 @@ def test_remove_add_remove_port_with_buffer_cfg(self, dvs, testlog):
 
         num = asic_db.wait_for_n_keys("ASIC_STATE:SAI_OBJECT_TYPE_PORT",
                                       num_of_ports-1,
-                                      polling_config = PollingConfig(polling_interval = 1, timeout = 5.00, strict = False))
+                                      polling_config = PollingConfig(polling_interval = 1, timeout = 30.00, strict = False))
```

Comment:
Why is the sleep timeout increased 6 fold here? Please avoid such large sleep intervals. I understand that this helps the test pass, but please narrow down the issue and fine-tune the sleep interval.